go.mod

```diff
@@ -24,6 +23,7 @@ require (
 
 require (
 	github.com/BurntSushi/toml v1.1.0 // indirect
+	github.com/CeresDB/ceresdbproto/golang v0.0.0-20221115062517-8cd021e19e44 // indirect
```

Comment:
This dependency should move to previous require same as before, because this dependency is not an indirect dependency.